code/game/objects/items/spear.dm

```diff
@@ -19,16 +19,26 @@
 	sharpness = SHARP_EDGED // i know the whole point of spears is that they're pointy, but edged is more devastating at the moment so
 	max_integrity = 200
 	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, BIO = 0, FIRE = 50, ACID = 30)
-	var/war_cry = "AAAAARGH!!!"
-	var/icon_prefix = "spearglass"
 	wound_bonus = -15
 	bare_wound_bonus = 15
+	/// For explosive spears, what we cry out when we use this to bap someone
+	var/war_cry = "AAAAARGH!!!"
+	/// The icon prefix for this flavor of spear
+	var/icon_prefix = "spearglass"
+	/// How much damage to do unwielded
+	var/force_unwielded = 10
```

Comment:
When you use a new variable like force_unwielded, it still initializes with force. So if you create a spear its initial value will be force and once you unwield it, it will become force_unwielded. Maybe you could use force_unwielded=force instead?

Reply:
force is set in `Initialize()` a few lines down